che-theia-plugins.yaml

```diff
@@ -200,11 +200,13 @@ plugins:
       cpuRequest: 30m
     extensions:
       - >-
-        https://download.jboss.org/jbosstools/vscode/stable/vscode-camelk/vscode-camelk-0.0.21-109.vsix
+        https://download.jboss.org/jbosstools/vscode/stable/vscode-camelk/vscode-camelk-0.0.22-135.vsix
       - >-
         https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-kubernetes-tools/vscode-kubernetes-tools-1.2.1.vsix
       - >-
         https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-yaml/vscode-yaml-0.11.1.vsix
+      - >-
+        https://open-vsx.org/api/redhat/vscode-commons/0.0.4/file/redhat.vscode-commons-0.0.4.vsix
```

Comment:
not available on download.jboss.org

Reply:
it is available on snapshots https://download.jboss.org/jbosstools/snapshots/vscode-commons//vscode-commons-0.0.4-44.vsix